submissions/BukkyOyetimehin/domManipulation/signUPForm.js

```diff
@@ -0,0 +1,153 @@
+let signupForm = document.querySelector('#validateForm'),
+    firstName = document.querySelector('#fname'),
+    lastName = document.querySelector('#lname'),
+    otherNames = document.querySelector('#onames'),
+    phoneNumber = document.querySelector('#phone'),
+    emailAddress = document.querySelector('#email'),
+    password = document.querySelector('#pwd'),
+    confirmPassword = document.querySelector('#cpwd'),
+    checkbox = document.querySelector('#terms'),
+    error = document.querySelector('.error-block');
+//First of all, the form must be checked to make sure all the mandatory fields are filled in.
+function required(field, event) {
```

Comment:
You only need to write on required function

Reply:
Hello Ada,
I don't know how to go about that. Please put me through.
Thank you.

Reply:
Why not make this function accept field name as a parameter, instead of having to repeat yourself to check for each field name